org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/server/UriExtensionsTest.xtend

```diff
@@ -270,7 +270,7 @@ class UriExtensionsTest {
 		var directory = createTempDir('some name with whitespaces').toFile;
 		assertTrue(directory.exists)
 		assertTrue(directory.directory)
-		assertTrue(CharMatcher.whitespace.matchesAnyOf(directory.name));
+		assertTrue(directory.name.contains(' '));
```

Comment:
why change this?

Reply:
My TP did not contain a Guava version with that method after I re-oomphed.

Reply:
test should be gradle oomphed and that one should use a newer guava

Reply:
So this is to be rejected?